README.md

```diff
@@ -12,3 +12,19 @@ A super minimal dashboard which lists each available site for all parked paths a
 E.g. `git clone https://github.com/aaemnnosttv/valet-dashboard.git dashboard`
 1. Move the new directory into a Valet-parked path or run `valet link` within it
 1. Run `valet open` from within the directory!
+
+## Setup as default site
+If you would like to access your dashboard by going to http://localhost,
+you have to adjust the Valet config under `~/config/valet/config.json`.
```

Comment:
* You have a typo in the path here: `config` should be `.config`
* Also, older installs may be using a different path to the config file, we should list both.

https://github.com/aaemnnosttv/valet-dashboard/blob/58a03f56b00f671348d50b42ee747a369f308be0/index.php#L2-L3